recipes/r-ensr/build.sh

```diff
@@ -1,22 +1,8 @@
-#!/bin/bash
```

Comment:
That was one too many, the shebang line can/should stay of course ;-)

Reply:
Whoops! I wanted to go to fast, haha.